drivers/i2c/hts21.py

```diff
@@ -0,0 +1,83 @@
+"""
+HTS21 - STMicro I2C temperature and humidity sensor driver for MicroPython
+
+Example usage:
+>>> from pyb import I2C
+>>> from hts21 import HTS21
+>>> hts = HTS21(I2C(1, I2C.MASTER), 0x5f)
+>>> hts.read_temp()
+22.06582
+>>> hts.read_humidity()
+57.44328
+"""
+from pyb import I2C
+import pyb
+import array
+
+class HTS21:    
+    HTS_HUMID_OUT = const(0x28)
+    HTS_CAL_H0x2 = const(0x30)
+    HTS_CAL_H1x2 = const(0x31)
+    HTS_CAL_H0T0 = const(0x36)
+    HTS_CAL_H1T0 = const(0x3a)
+
+    HTS_TEMP_OUT = const(0x2a)
+    HTS_CAL_T0DEGx8 = const(0x32)
+    HTS_CAL_T1DEGx8 = const(0x33)
+    HTS_CAL_T0 = const(0x3c)
+    HTS_CAL_T1 = const(0x3e)
+    
+    def __init__(self, i2c, address, fixed_point = False):
+        """
+        `fixed_point` makes this class use fixed point return values with 2 decimal places 
+        (for hw without floating point support)
+        """
+        self.i2c = i2c
+        self.address = address
+        self.fixed_point = fixed_point
+        self.hts_calib = array.array("h",[0,0,0,0,0,0,0,0,0,0])
```

Comment:
I wanted to make sure it a signed 16bit integer list. It's used/filled by the i2c.memread command and `int.from_bytes` (which does not yet support the `signed` kw..). 

I thought using a list of Ints would have forced me to do some expensive conversions (using the struct package for example)